src/uwtools/config/formats/yaml.py

```diff
@@ -5,7 +5,13 @@
 import yaml
 
 from uwtools.config.formats.base import Config
-from uwtools.config.support import INCLUDE_TAG, TaggedString, add_yaml_representers, log_and_error
+from uwtools.config.support import (
```

Comment:
I always get wary when a `from-include` block starts to need a new line. I suspect if we have to pack more things in `config.support`, we'll be better off just importing the whole thing or splitting it up further